bcgov/biohubbc
# BHBC-2306: Hide Submit functionality

# Overview

## Links to Jira tickets

[BHBC-2306](https://quartech.atlassian.net/browse/BHBC-2306?atlOrigin=eyJpIjoiZjM3NjAwOTc0ZTY0NGUxMGFmNTNiYTEwMDM1YTk2ZWMiLCJwIjoiaiJ9)

## Description of relevant changes

- Added System Role Guard to various status columns and submission buttons
- Added `TODO: PRODUCTION_BANDAGE` to the pages to mark what changes to be removed later

## Testing Steps
- Login as a Data/ System admin
- Create a project and survey
- Add documents/ observations to the Survey and see submit buttons and status on documents
- Add documents to project and see submission status and submit button
- Modify current permissions to be a Creator (this will throw an error but can be ignored for now. Issue tracked in [BHBC-2309](https://quartech.atlassian.net/browse/BHBC-2309?atlOrigin=eyJpIjoiNTZiNDZmOWM5ZTRkNDU2M2IxNDFjNzY5ZThmMjEyMmIiLCJwIjoiaiJ9)
- Step through newly created project/ survey, all status and submission information will now be hidden

## PR Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### Code


### Style


### Documentation


### Tests


### Linting/Formatting

       _See the `lint` commands in package.json_
       _See the `format` commands in package.json_

### SonarCloud
